src/views/token-swap/components/selected-quote/selected-quote.view.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import useSelectedQuoteStyles from './selected-quote.style'
+import Spinner from '../../../shared/spinner/spinner.view'
+import {
+  getTokenAmountInPreferredCurrency,
+  getFixedTokenAmount
+  , CurrencySymbol
+} from '../../../../utils/currencies'
+import { MAX_TOKEN_DECIMALS } from '../../../../constants'
+
+function SelectedQuote ({
+  quotes,
+  fiatExchangeRates,
+  preferredCurrency,
+  selectedTokens,
+  selectedLpId,
+  onGoToQuotes,
+  onOpenOfferSidenav
+}) {
+  const classes = useSelectedQuoteStyles()
+  const isBestQuote = selectedLpId === 'best'
+  const isLoading = quotes.status === 'loading'
+  const successful = quotes.status === 'successful'
+
+  const [quote, setQuote] = React.useState({ rate: 0 })
+  const [fiatReward, setFiatReward] = React.useState(0)
+  const [reward, setReward] = React.useState(0)
+
+  React.useEffect(() => {
+    if (quotes.status === 'successful' && quotes.data) {
```

Comment:
can't you use isSuccessful?

Reply:
Elias told me something about typescript and the future refactor.

Reply:
but I can remove it because if data exists status should be successful

Reply:
Yep, that should be as it's @AlbertoElias. In fact, what I don't like is to have this `isSuccessful`, `isFailure` consts at the top hahaha. This has to do with TypeScript, as you need to be explicit on what you are checking and it's better to be prepared for the change